pkg/kubectl/cmd/cp/cp.go

```diff
@@ -143,6 +148,8 @@ func extractFileSpec(arg string) (fileSpec, error) {
 
 // Complete completes all the required options
 func (o *CopyOptions) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
+	o.cmd = cmd
```

Comment:
rather than holding a pointer to the command, consider storing the name of the parent command here (e.g. `o.ExecParentCmdName = cmd.Parent().Name()`

then, make the hint about using exec condition on ExecParentCmdName being non-empty. that way, CopyOptions stays reusable